stancon_2020/rise.css

```diff
@@ -0,0 +1,322 @@
+
+:root {
+  --xr-font-color0: var(--jp-content-font-color0, rgba(0, 0, 0, 1));
+  --xr-font-color2: var(--jp-content-font-color2, rgba(0, 0, 0, 0.54));
+  --xr-font-color3: var(--jp-content-font-color3, rgba(0, 0, 0, 0.38));
+  --xr-border-color: var(--jp-border-color2, #e0e0e0);
+  --xr-disabled-color: var(--jp-layout-color3, #bdbdbd);
+  --xr-background-color: var(--jp-layout-color0, white);
+  --xr-background-color-row-even: var(--jp-layout-color1, white);
+  --xr-background-color-row-odd: var(--jp-layout-color2, #eeeeee);
+}
+
+
+.xr-wrap {
+  display: block!important;
+  min-width: 300px!important;
+  max-width: 700px!important;
+}
+
+.xr-text-repr-fallback {
+  /* fallback to plain text repr when CSS is not injected (untrusted notebook) */
+  display: none!important;
+}
+
+.xr-header {
+  padding-top: 6px!important;
+  padding-bottom: 6px!important;
+  margin-bottom: 4px!important;
+  border-bottom: solid 1px var(--xr-border-color);
+}
+
+.xr-header > div,
+.xr-header > ul {
+  display: inline!important;
+  margin-top: 0!important;
+  margin-bottom: 0!important;
+}
+
+.xr-obj-type,
+.xr-array-name {
+  margin-left: 2px!important;
+  margin-right: 10px!important;
+}
+
+.xr-obj-type {
+  color: var(--xr-font-color2);
+}
+
+.xr-sections {
+  padding-left: 0 !important;
+  display: grid!important;
+  /* Make grid larger so for xarray repr thing like posterior predictive dont wrap*/
+  grid-template-columns: 250px auto auto 1fr 20px 20px;
```

Comment:
This is fixing group names (i.e. posterior predictive) but breaks xarray dataset view. I think we have to fix https://github.com/arviz-devs/arviz/issues/1248 to get it to acctually work.

Reply:
You're right I changed it back